recipes/cyrus-sasl/all/conanfile.py

```diff
@@ -143,33 +144,45 @@ def _configure_autotools(self):
             "--with-mysql={}".format(rootpath_no(self.options.with_mysql, "libmysqlclient")),
             "--without-sqlite",
             "--with-sqlite3={}".format(rootpath_no(self.options.with_sqlite3, "sqlite3")),
-        ]
+        ])
         if self.options.with_gssapi:
-            args.append("--with-gss_impl=mit")
+            tc.configure_args.append("--with-gss_impl=mit")
+        tc.generate()
+
+        deps = AutotoolsDeps(self)
+        deps.generate()
+        if not cross_building(self):
+            env = VirtualRunEnv(self)
+            env.generate(scope="build")
 
-        autotools.configure(configure_dir=self._source_subfolder, args=args)
-        return autotools
+    def _patch_sources(self):
+        shutil.copy(self._user_info_build["gnu-config"].CONFIG_SUB,
```

Comment:
can the new copy now be used here?

Reply:
Didn't you already ask in another PR? The answer is no.

Reply:
![image](https://user-images.githubusercontent.com/16867443/195963862-029787c6-8aa5-4741-8d99-19bd631e4a95.png)


![image](https://user-images.githubusercontent.com/16867443/195963828-29257be5-70e3-4280-84e3-27a5b633b236.png)


Yeah sorry but I'll probably ask again 😅 since it looks out of place and in a few hundred PR reviews I'll forget